src/tribol/geom/GeomUtilities.cpp

```diff
@@ -574,28 +594,29 @@ FaceGeomError Intersection2DPolygon( const real* const RESTRICT xA,
 
       if (!orientA || !orientB)
       {
-         SLIC_DEBUG( "Intersection2DPolygon(): check face orientations for face A." );
+        //  SLIC_DEBUG( "Intersection2DPolygon(): check face orientations for face A." );
```

Comment:
Do we want to guard all SLIC_DEBUG statements in this routine (and elsewhere) with TRIBOL_USE_HOST? I'm noticing it is done in some locations, but not others.